File: src/main/java/hacker/solution/Add.java
package hacker.solution;

class Add {
    public int add(int... values) {
        int sum=0;
        int i=0;
        for (int value: values) {
            if (i > 0) System.out.print("+");
            System.out.print(value);
            sum += value;
            i++;
        }
        System.out.println("="+sum);
        return sum;
    }
}

File: src/main/kotlin/hacker/solutionk/Add.kt
package hacker.solutionk

class Add {
    fun add(vararg values: Int): Int {
        var sum = 0
        var i = 0
        for (value in values) {
            if (i > 0) print("+")
            print(value)
            sum += value
            i++
        }
        println("=$sum")
        return sum
    }
}
